lib/index.js

```diff
@@ -61,6 +61,10 @@ Object.defineProperties(Video, {
     enumerable: true,
     value: require('./vendor/loglevel')
   },
+  runPreflight: {
+    enumerable: true,
+    value: require('../es5/preflight').runPreflight
```

Comment:
@charliesantos, I see that here we are 1st compiling `.ts` files into es5 and then using it from here. I do not like this dependency on generated code from the non-generated code. I am afraid that as we write more and more typescript code, our local workflow would become cumbersome as it would require `build` process. 
I see that this is needed because this file is `.js` and thus cannot include `.ts`. Would converting this file into `.ts` solve this problem or is there more to it?

Reply:
Hey @makarandp0 , good question. Actually, this was originally using the ts file. The problem is that this causes an issue with our integration tests. karma-typescript or browserify is having an issue requiring both ts and js. See PR description regarding the problem on integration tests.

Reply:
@charliesantos What if we convert the whole `index.js` to `index.ts`. The JS modules can be imported as:
```ts
import connect = require('./connect');
```
and the TS modules (only preflight for now) can be imported as:
```ts
import { runPreflight } from './preflight';
```